ietf/templates/api/index.html

```diff
@@ -284,7 +284,7 @@ <h2 id="signing-keys">
         <p>
             When sending notifications to other APIs, the datatracker may sign
             information with a
-            <a href="https://datatracker.ietf.org/doc/html/rfc7515">
+            <a href="/doc/html/rfc7515">
```

Comment:
All of these should instead be changed to use the url resolver, e.g. `<a href="{% url 'ietf.doc.views_doc.document_html' name='rfc7515' %}>`

Reply:
There are many more that don't use the URL resolver, which is why I made the change in this way - want me to change those as well?

Reply:
Yes, we should move away from having the templates assume they know what the URL structure looks like.